apis/Google.Storage.V1/Google.Storage.V1.IntegrationTests/StorageFixture.cs

```diff
@@ -97,9 +97,9 @@ public sealed class StorageFixture : IDisposable, ICollectionFixture<StorageFixt
         public HttpClient HttpClient { get; } = new HttpClient();
 
         /// <summary>
-        /// Gets the path to a service account credentials JSON file.
+        /// Gets a <see cref="UrlSigner"/> instance which can be used for testing.
         /// </summary>
-        public string CredentialsFilePath { get; } = GetCredentialsFilePath();
+        public UrlSigner UrlSigner { get; } = new UrlSigner(GetCredentialsFilePath());
```

Comment:
I'd change this into a method - it's a bit weird for a property to create a new instance each time (that reads a file).